### DISCUSSION TITLE
abp generate-proxy -t csharp -u http://127.0.0.1:12555 have error

### DISCUSSION THREAD

**Discussion by tuohaibei** [abpframework/abp#17934] [Q&A]
could any one could help me？
https://stackoverflow.com/questions/77321978/abp-generate-proxy-t-csharp-u-http-127-0-0-112555-have-error

**Comment by maliming** [comment]
hi

`The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing`

Please check the URL(`http://127.0.0.1:12555/`).
